frontend/src/views/OtherUses/OtherUsesSummary.jsx

```diff
@@ -0,0 +1,152 @@
+
+  const { data: currentUser } = useCurrentUser()
+  const userRoles = currentUser?.roles?.map((role) => role.name) || []
+  const isAuthorized = [
+    roles.analyst,
+    roles.compliance_manager,
+    roles.director,
+    roles.supplier
+  ].some((role) => userRoles.includes(role))
+
+  const gridRef = useRef()
+  const apiService = useApiService()
+  const { t } = useTranslation(['common', 'otherUses'])
+  const navigate = useNavigate()
+  const location = useLocation()
+
+  useEffect(() => {
+    if (location.state?.message) {
+      setAlertMessage(location.state.message)
+      setAlertSeverity(location.state.severity || 'info')
+    }
+  }, [location.state])
+
+  const gridOptions = useMemo(() => ({
+    overlayNoRowsTemplate: t('otherUses:noOtherUsesFound'),
+    autoSizeStrategy: {
+      type: 'fitCellContents',
+      defaultMinWidth: 50,
+      defaultMaxWidth: 600
+    }
+  }), [t])
+
+  const columns = [
+    { headerName: "Fuel Type", field: "fuelType", floatingFilter: false },
+    { headerName: "Fuel Category", field: "fuelCategory", floatingFilter: false },
+    { headerName: "Quantity Supplied", field: "quantitySupplied", floatingFilter: false },
+    { headerName: "Units", field: "units", floatingFilter: false },
+    { headerName: "Expected Use", field: "expectedUse", floatingFilter: false },
+    { headerName: "Rationale", field: "rationale", floatingFilter: false },
+  ];
+
+  const getRowId = (params) => params.data.otherUsesId
+
+  const handleGridKey = () => setGridKey(`other-uses-grid-${uuid()}`)
+
+  const handleRowClicked = (params) => {
+    if (!isAuthorized) return
+    navigate(
+      ROUTES.REPORTS_ADD_OTHER_USE_FUELS.replace(':otherUsesID', params.data.otherUsesId)
+    )
+  }
+
+  const handleDownloadOtherUses = async () => {
+    setIsDownloadingOtherUses(true)
+    setAlertMessage('')
+    try {
+      await apiService.download(ROUTES.ADMIN_OTHER_USES + '/export')
+      setIsDownloadingOtherUses(false)
+    } catch (error) {
+      console.error('Error downloading other uses information:', error)
+      setIsDownloadingOtherUses(false)
+      setAlertMessage(t('otherUses:otherUsesDownloadFailMsg'))
+      setAlertSeverity('error')
+    }
+  }
+
+  return (
+    <Grid2 className="other-uses-container" mx={-1}>
+      <div>
+        {alertMessage && (
+          <BCAlert data-test="alert-box" severity={alertSeverity}>
+            {alertMessage}
+          </BCAlert>
+        )}
+      </div>
+
+      <Stack
+        direction={{ md: 'column', lg: 'row' }}
+        spacing={{ xs: 2, sm: 2, md: 3 }}
+        useFlexGap
+        flexWrap="wrap"
+        mb={2}
+      >
+        <Typography variant="h5" color="primary" data-test="title">
+          {t('otherUses:summaryTitle')}
+        </Typography>
+        <FontAwesomeIcon 
+          icon={faEdit} 
+          size={'lg'}
+          onClick={() => navigate(
+            ROUTES.REPORTS_ADD_OTHER_USE_FUELS.replace(
```

Comment:
There seems to be a mismatch in the route links in ActivityLinksList for the main compliance report link lists on line 42.